SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialWorkerConnection.cpp

```diff
@@ -125,12 +136,51 @@ void USpatialWorkerConnection::OnLoginTokens(void* UserData, const Worker_Alpha_
 			FString DeploymentName = FString(LoginTokens->login_tokens[i].deployment_name);
 			if (DeploymentToConnect.Compare(DeploymentName) == 0)
 			{
-				Connection->LocatorConfig.LoginToken = FString(LoginTokens->login_tokens[i].login_token);
+				LocatorConfig.LoginToken = FString(LoginTokens->login_tokens[i].login_token);
 				break;
 			}
 		}
 	}
-	Connection->ConnectToLocator();
+	ConnectToLocator();
+}
+
+bool USpatialWorkerConnection::TrySetupConnectionConfigFromCommandLine(const FString& SpatialWorkerType)
```

Comment:
that function doesn't get called anywhere in the code you changed here. Does it still exist in the SpatialNetDriver? (not sure in which state the 0.8.1 rc is, so not sure whether the changes that we made on the NetDriver & the WorkerConnection are all in)

Reply:
This is called in ExampleProjectCode changes: 
https://github.com/spatialos/UnrealGDKExampleProject/pull/108/files

Reply:
All extra changes were made due to the code change in the example project.

Reply:
In fact all the extra changes were done just to facilitate calling TrySetupConnectionConfigFromCommandLine from exampleproject.

Reply:
I am not sure whether example project can work with the PR. If `SpatialWorkerConnection` can get correct dev auth token from command line or unreal editor setting, maybe it could work, but that should be verified first.